packages/trees/frontend/sirius-components-trees/src/modals/upload-document/UploadDocumentModalMachine.ts

```diff
@@ -0,0 +1,154 @@
+/*******************************************************************************
```

Comment:
Remove the original in `packages/core/frontend/sirius-components/src/modals/upload-document`.